### DISCUSSION TITLE
Experimental support for Kitty graphics protocol should be flagged as "Linux/Mac" only

### DISCUSSION THREAD

**Discussion by jeansagi** [junegunn/fzf#3482] [General]
As Kitty is not ported to Windows yet, the Kitty graphics protocol **fzf 0.43.0** supports should be clearly advertised as Linux/Mac only so clueless Windows users (like me) don't try this option! 

;)

J.

**Comment by kovidgoyal** [comment]
The kitty graphics protocol is not implemented only in kitty. It exists, for instance, at least partially in wezterm which runs on windows.

**Comment by junegunn** [comment]
Good point. I updated the release note.

The way fzf implements the Kitty graphics protocol is to simply pass through regions of text that correspond to the graphics data to the underlying terminal.

https://github.com/junegunn/fzf/blob/d51b71ee8039df8f51bd348b43c68ee462d4a496/src/tui/light.go#L34-L37

However, if I'm not mistaken, the tcell library we use for the Windows build does not provide such a "pass-through" mechanism, so it is not currently supported on the fzf side.

https://github.com/junegunn/fzf/blob/d51b71ee8039df8f51bd348b43c68ee462d4a496/src/tui/tcell.go#L101-L104

**Reply by jeansagi** [reply]
Obviously it depends on several factors, but it would be awesome to be able to view images in **fzf** on **Windows**.

And BTW, Using **Kitty** on Windows would be awesome too.

<br>

J.

**Comment by junegunn** [comment]
With 68db9cb499ab32190edae6c285942c5fb7cf39ed, it will work on Windows as well (though I haven't actually tested it on Windows).

**Reply by junegunn** [reply]
Nope. https://github.com/junegunn/fzf/commit/38e3694d1ca5f138f26fbbffb7c0122b58eee8dc
